### PR TITLE
Added quick command to add tasks as if they were entered in the todoist UI

### PR DESCRIPTION
**Pull request by Lindenk**
Todoist has a way to allow the user to send a string and have it parsed similar to how it would be in the UI. This adds a command to the cli for it.

### CODE REVIEW

File: lib/todoist.go [hunk 1]
```diff
@@ -94,6 +94,16 @@ func (c *Client) ExecCommands(ctx context.Context, commands Commands) error {
...
 }
 
+func (c *Client) QuickCommand(ctx context.Context, text string) error {
+	var r ExecResult
+
+	values := url.Values{
+		"text": {text},
+	}
+
+	return c.doApi(ctx, http.MethodPost, "quick/add", values, &r)
```
Comment (sachaos) [inline lib/todoist.go]: Please sync.
https://github.com/sachaos/todoist/blob/master/add.go#L42

**Comment by Lindenk** [comment]
Wow, snappy response! I added a sync call at the end of the command.

**Comment by sachaos** [comment]
@Lindenk Thank yo for your PR! 🎉

**Comment by Lindenk** [comment]
No problem! I just realized I forgot to add a line to the readme for the new command though. Sorry!

**Pull request merged**